sample-android-compose/src/main/java/com/mobilejazz/kmmsample/mvi/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.mobilejazz.kmmsample.mvi
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.mobilejazz.kmmsample.mvi.application.ui.screens.NavGraphs
+import com.mobilejazz.kmmsample.mvi.application.ui.screens.hackerPosts.HackerPostsScreen
+import com.mobilejazz.kmmsample.mvi.ui.theme.HarmonykotlinTheme
+import com.ramcosta.composedestinations.DestinationsNavHost
+
+class MainActivity : ComponentActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContent {
+      HarmonykotlinTheme {
```

Comment:
The `kotlin` word should be upercase. Instead of `HarmonykotlinTheme` should be `HarmonyKotlinTheme`